### ISSUE TITLE
Difficult Install

### ISSUE THREAD

**Issue by zQueal** [philipl/pifs#3]
Installation steps as follows:

```
git clone git@github.com:philipl/pifs.git
cd pifs; ./autogen.sh
```

Errors during autogen:

![d1vpfa1](https://f.cloud.github.com/assets/239193/952261/7a71470e-03c7-11e3-9db4-27a99df0440b.png)

Any ideas?

**Comment by NginUS** [comment]
Running Debain jessie in Virtualbox:

I got this error when running ./configure.

./configure: line 4298: syntax error near unexpected token `FUSE,'
./configure: line 4298: `PKG_CHECK_MODULES(FUSE, fuse >= 2.8)'

I installed pkg-config, fuse, libfuse-dev to no avail.

./configure wouldn't complete until I commented out line 4298 in 'pifs/configure' entirely.

I've yet to see if it mattered, will soon find out.
